integration/kubernetes/alluxio-fuse-client.yaml.template

```diff
@@ -17,24 +17,24 @@ metadata:
   name: alluxio-fuse-client
   labels:
     app: alluxio
-    chart: alluxio-0.5.0
-    release: release-name
+    chart: alluxio-0.5.1
+    release: RELEASE-NAME
```

Comment:
Do we need to change the release name? I don't have a preference between lower or upper case though.

Reply:
@cheyang Seems like helm might choose a different string based on version. So I've explicitly defined the release name in https://github.com/Alluxio/alluxio/pull/10245. Please rebase and run generate again. Thx.